src/client.cpp

```diff
@@ -948,9 +950,11 @@ void writePlayerPos(LocalPlayer *myplayer, NetworkPacket *pkt)
 		[12+12] s32 pitch*100
 		[12+12+4] s32 yaw*100
 		[12+12+4+4] u32 keyPressed
+		[12+12+4+4+4] s32 fov*100
+		[12+12+4+4+4+4] s32 v_range
 	*/
-
 	*pkt << position << speed << pitch << yaw << keyPressed;
+	*pkt << fov << v_range;
```

Comment:
Can I do this and not break wire compatibility? I think a newer server can work with an older client, but not necessarily a new client with an old server - the server would not drain the packet... Not sure.